server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -289,13 +289,55 @@ export function configureWebSocketServices(
 			if (throttleErrorPerTenant) {
 				return Promise.reject(throttleErrorPerTenant);
 			}
+
```

Comment:
remove the extra line :)

Reply:
We can add this line to separate the logic for throttling, and the below parts.